### PR TITLE
translation content/zh/docs/reference/setup-tools/kubeadm/ kubeadm-jo…

### PR DESCRIPTION
**Pull request by tanjunchen**

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join-phase.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+weight: 90
+---
+-->
+
+<!--
+`kubeadm join phase` enables you to invoke atomic steps of the join process.
+Hence, you can let kubeadm do some of the work and you can fill in the gaps
+if you wish to apply customization.
+-->
+`kubeadm join phase` 使您能够调用 `join` 过程的基本原子步骤。因此，如果希望自定义应用，则可以让 kubeadm 做一些工作，然后填补空白。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join-phase.md]:
```suggestion
`kubeadm join phase` 使您能够调用 `join` 过程的基本原子步骤。因此，如果希望执行自定义操作，可以让 kubeadm 做一些工作，然后由用户来补足剩余操作。
```

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join-phase.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+{{< tabs name="tab-phase" >}}
+{{< tab name="phase" include="generated/kubeadm_join_phase.md" />}}
+{{< /tabs >}}
+
+## kubeadm join 启动前检查阶段 {#cmd-join-phase-preflight}
+
+<!--
+Using this phase you can execute preflight checks on a joining node.
+-->
+使用此命令可以在 node 节点上执行启动前检查。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join-phase.md]:
```suggestion
使用此命令可以在即将加入集群的节点上执行启动前检查。
```

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-reset-phase.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+weight: 90
+---
+-->
+
+<!--
+`kubeadm reset phase` enables you to invoke atomic steps of the node reset process.
+Hence, you can let kubeadm do some of the work and you can fill in the gaps
+if you wish to apply customization.
+-->
+`kubeadm reset phase` 使您能够调用 `reset` 过程的基本原子步骤。因此，如果希望自定义应用，则可以让 kubeadm 做一些工作，然后填补空白。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-reset-phase.md]: 同前

Comment (tanjunchen) [reply content/zh/docs/reference/setup-tools/kubeadm/kubeadm-reset-phase.md]: done.

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**